lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3137,7 +3137,7 @@ namespace Js
 
         } autoRestore(this->GetThreadContext());
 
-        if (!Js::Configuration::Global.EnableJitInDebugMode())
+        if (Js::Configuration::Global.EnableJitInDebugMode())
```

Comment:
>if (Js::Configuration::Global.EnableJitInDebugMode()) [](start = 8, length = 53)

Sorry I didn't understood the logic. Are you saying we disable jit in debug mode?

Run below test in full with "-mic:1 -off:simplejit Test.js -targeted" and you will see the difference

function run() {
  foo();
  foo();
}
function foo() {
  WScript.Echo(Debug.isInJit());
}
run();
WScript.Attach(run);